SPEC.md

```diff
@@ -5915,25 +5946,28 @@ Example input:
 
 ```json
 {
-  "hisat2.index_tar_gz": "https://genome-idx.s3.amazonaws.com/hisat/grch38_genome.tar.gz",
-  "hisat2.sra_acc": "SRR3440404",
-  "hisat2.max_reads": 10
+  "index_tar_gz": "https://genome-idx.s3.amazonaws.com/hisat/grch38_genome.tar.gz",
+  "fastq1": "https://storage.googleapis.com/genomics-public-data/gatk-examples/example1/NA20274/ERR250968_1.filt.fastq.gz",
```

Comment:
I would say let's use our generated FASTQ files here so that we can eliminate the reliance on external data sources. The fact that these FASTQ files won't align well is irrelevant.

Reply:
I think it depends on what we are trying to test here. For now I skip this test in github actions. But I think the point it more to provide genuine examples of how to run complex tasks with real data.

Reply:
Yep going to subset and push.

Reply:
My idea would be lets put this into a new issue and remove in 1.3 and keep these here as optional for 1.1 and 1.2

Reply:
@claymcleod  made a new issue to discuss: #730

Reply:
Clay this is reverted to the changes needed to get this test to be working.